### PR TITLE
Move elm to own repo

### PR DESCRIPTION
**Pull request by zcei**
You can find it here: [webpack-blocks/elm](https://github.com/webpack-blocks/elm)

Related PR with the e2e test: https://github.com/webpack-blocks/elm/pull/1

### CODE REVIEW

**Comment by zcei** [comment]
@vlad-zhukov I think I just decreased build time from >20min to ~5min 😀

File: yarn.lock [hunk 1]
```diff
@@ -52,6 +52,14 @@
   version "15.6.15"
   resolved "https://registry.yarnpkg.com/@types/react/-/react-15.6.15.tgz#1856f932120311aa566f91e6d0c6e613d6448236"
 
+"@webpack-blocks/elm@^1.0.0":
+  version "1.0.0"
```
Comment (vlad-zhukov) [inline yarn.lock]: Where did it come from? It shouldn't be here.

**Comment by dmitmel** [comment]
Awesome!

**Comment by jvanbruegge** [comment]
The built time is quite amazing

**Comment by farism** [comment]
If elm was experiencing slow build times, it may have been due to invalid reporting of cores to the elm compiler which severely slows down the compiling of modules https://github.com/elm-lang/elm-compiler/issues/1473

File: README.md [hunk 2]
```diff
@@ -182,7 +182,7 @@ Check out the [sample app](./packages/sample-app) to see a webpack config in act
 - [assets](./packages/assets)
 - [babel](./packages/babel)
 - [dev-server](./packages/dev-server)
-- [elm](./packages/elm)
+- [elm](https://github.com/webpack-blocks/elm)
```
Comment (andywer) [inline README.md]: 👍

**Pull request merged**